keyboards/rot13labs/h4ckb0ard/info.json

```
@@ -0,0 +1,159 @@
+{
+    "manufacturer": "rot13labs",
+    "keyboard_name": "H4CKB0ARD",
+    "maintainer": "c0ldbru",
+    "bootloader": "usbasploader",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": false,
```

This is preferred to be enabled if possible. I believe there's enough endpoints.
```suggestion
        "mousekey": true,
```